### PR TITLE
support angle_controller debug

### PR DESCRIPTION
**Pull request by ctzsnooze**
Support for the new `ANGLE_CONTROLLER` debug from @ChrisRosser.

This is for testing purposes only, at this time.

See firmware PR [#12041](https://github.com/betaflight/betaflight/pull/12041).

EDIT: [#12231](https://github.com/betaflight/betaflight/pull/12231).

### CODE REVIEW

**Review by sugaarK — APPROVED** [review]

**Review by haslinghuis — APPROVED** [review]

File: js/flightlog_fielddefs.js [hunk 1]
```diff
@@ -349,6 +349,7 @@ let
         "ATTITUDE",
         "VTX_MSP",
         "GPS_DOP",
+        "ANGLE_CONTROLLER",
```
Comment (haslinghuis) [inline js/flightlog_fielddefs.js:352]:
```suggestion
        "ANGLE_FEEDFORWARD",
```

File: js/flightlog_fields_presenter.js [hunk 2]
```diff
@@ -690,6 +690,13 @@ function FlightLogFieldPresenter() {
             'debug[2]': 'hDOP (horizontal - 2D)',
             'debug[3]': 'vDOP (vertical - 1D)',
         },
+        'ANGLE_CONTROLLER' : {
```
Comment (haslinghuis) [inline js/flightlog_fields_presenter.js:693]:
```suggestion
        'ANGLE_FEEDFORWARD' : {
```

File: js/flightlog_fields_presenter.js [hunk 3]
```diff
@@ -1354,6 +1361,16 @@ function FlightLogFieldPresenter() {
                         default:
                             return (value / 100).toFixed(2);
                     }
+                case 'ANGLE_CONTROLLER':
```
Comment (haslinghuis) [inline js/flightlog_fields_presenter.js:1364]:
```suggestion
                case 'ANGLE_FEEDFORWARD':
```

File: js/graph_config.js [hunk 4]
```diff
@@ -931,6 +931,27 @@ GraphConfig.load = function(config) {
                             default:
                                 return getCurveForMinMaxFields(fieldName);
                             }
+                    case 'ANGLE_CONTROLLER':
```
Comment (haslinghuis) [inline js/graph_config.js:934]:
```suggestion
                    case 'ANGLE_FEEDFORWARD':
```

**Review by haslinghuis — CHANGES_REQUESTED** [review]
`ANGLE_CONTROLLER` has been renamed to `FEEDFORWARD_ANGLE` in firmware.

Now it's `ANGLE_MODE` :)

**Review by nerdCopter — APPROVED** [review]

**Comment by haslinghuis** [comment]
Superseded by #674

**Pull request closed without merging**